ai_scientist/llm.py

```diff
@@ -199,6 +214,17 @@ def get_response_from_llm(
         )
         content = response.choices[0].message.content
         new_msg_history = new_msg_history + [{"role": "assistant", "content": content}]
+    elif "gemini" in model:
+        new_msg_history = msg_history + [{"role": "user", "parts": msg}]
+        response = client.generate_content(
```

Comment:
No system message!

Reply:
Yes, looks like Gemini doesn't have support for system messages as of now. This is what I found online: 
[Discussion1](https://www.googlecloudcommunity.com/gc/AI-ML/Implementing-System-Prompts-in-Gemini-Pro-for-Chatbot-Creation/m-p/715501), [Discussion2](https://github.com/langchain-ai/langchain/issues/14700)